### ISSUE TITLE
ls : display group information ?

### ISSUE THREAD

**Issue by elebeaup** [rmyorston/busybox-w32#33]
The ls command doesn't display group information (only user information).

```C:\Dev\test
λ ls -l
total 4
-rw-------    1 emmanuel 0                7 Feb 24 11:01 test
```

**Comment by rmyorston** [comment]
File permissions on Microsoft Windows are different from POSIX.  I'm not aware of any equivalent of groups in Windows.

**Comment by elebeaup** [comment]
Indeed, I have compared with ls command provided by msysgit. But, msysgit is a subset of msys (which is a fork of cygwin and cygwin provide POSIX functionality). So, it is quite normal than ls (provided by busybox-w32) doesn't display properly group information. Windows handle file permission with acl functionality but I think it is very difficult to implement it in busybox-w32.

Thanks for your great job.

**Comment by rmyorston** [comment]
I've tidied up uid/gid handling.  Previously different fake-POSIX routines were using different values: `getuid`/`getgid` used 1, `stat` used 0 and `getpwuid` used 1000.  I've standardised on 1000.

And I've added a fake `getgrgid` so `ls` shows a group name.  It's always just your user name, but that's better than nothing (or zero, which is what it was when `stat` returned a gid of 0).

Cygwin has a long write-up on [POSIX accounts, permission, and security] (https://cygwin.com/cygwin-ug-net/ntsec.html). I certainly don't plan to implement anything like that.
